PyInstaller/utils/hooks/qt.py

```diff
@@ -7,31 +7,40 @@
 # The full license is in the file COPYING.txt, distributed with this software.
 #
 # SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
+import glob
+import json
 import os
 import sys
-import json
-import glob
 
-from ..hooks import eval_statement, exec_statement, get_homebrew_path, \
-    get_module_file_attribute
-from PyInstaller.depend.bindepend import getImports, getfullnameof
+from PyInstaller.depend.bindepend import getfullnameof, getImports
+
 from ... import log as logging
-from ...compat import is_win, is_darwin, is_linux
+from ...compat import is_darwin, is_linux, is_win
 from ...utils import misc
+from ..hooks import (
+    eval_statement,
+    exec_statement,
+    get_homebrew_path,
+    get_module_file_attribute,
+)
 
 logger = logging.getLogger(__name__)
 
 
+Qt_Librairies = ['PyQt5', 'PySide2', 'PySide6']
```

Comment:
```suggestion
Qt_Libraries = ['PyQt5', 'PySide2', 'PySide6']
```

Reply:
Missed that one, will fix in the next commit